<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mainpage</class>
 <widget class="QWidget" name="Mainpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
   <property name="leftMargin">
    <number>64</number>
   </property>
   <property name="topMargin">
    <number>64</number>
   </property>
   <property name="rightMargin">
    <number>64</number>
   </property>
   <property name="bottomMargin">
    <number>64</number>
   </property>
   <item>
    <spacer name="verSpacerTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horLayoutRoles">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <spacer name="horSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnLeft">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
}
:hover {
	background: rgba(255,255,255,20);
}
:pressed {
	background: rgba(0,0,0,20);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../src.qrc">
         <normaloff>:/arrow/src/arrowLeft.png</normaloff>:/arrow/src/arrowLeft.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RoleDisplay" name="widgetRoles" native="true">
       <property name="minimumSize">
        <size>
         <width>282</width>
         <height>282</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
}
:hover {
	background: rgba(255,255,255,20);
}
:pressed {
	background: rgba(0,0,0,20);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../src.qrc">
         <normaloff>:/arrow/src/arrowRight.png</normaloff>:/arrow/src/arrowRight.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>71</width>
         <height>282</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verSpacerMid">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horLayoutEdit" stretch="1,5,1">
     <item>
      <spacer name="horSpacerEditLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>24</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>你的名称</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelServ">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>服务器</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editServ">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horLayoutEnter">
         <item>
          <spacer name="horSpacerEnter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnEnter">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	color: #FFFFFF;
	background-color: rgb(20,196,188);
	border: 2px solid rgb(24, 204, 192);
	padding: 6px 30px;
}

:hover{
	background-color: rgb(22,218,208);
}
:pressed{
	background-color: rgb(17,171,164);
}</string>
           </property>
           <property name="text">
            <string>进入</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horSpacerEditRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verSpacerBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoleDisplay</class>
   <extends>QWidget</extends>
   <header>Mainpage/roledisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../src.qrc"/>
 </resources>
 <connections/>
</ui>
